examples/production/scc-spire-spiffe-csi-driver.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: security.openshift.io/v1
```

Comment:
Is this file redundant because of Is this file redundant because of charts/spire/charts/spiffe-csi-driver/templates/scc-spire-spiffe-csi-driver.yaml?